### PR TITLE
added gazebo 9 compatibility

### PR DESCRIPTION
**Pull request by unklar**
Added gazebo 9 compatibility with regard to ros melodic.

### CODE REVIEW

File: src/gazebo_ros_diffdrive_uos.cpp [hunk 1]
```diff
@@ -181,7 +181,11 @@ void GazeboRosDiffdrive::Load(physics::ModelPtr _parent, sdf::ElementPtr _sdf )
     max_velocity_ = _sdf->GetElement("max_velocity")->Get<double>();
 
   //initialize time and odometry position
-  prev_update_time_ = last_cmd_vel_time_ = this->my_world_->GetSimTime();
+#if GAZEBO_MAJOR_VERSION > 8
+    prev_update_time_ = last_cmd_vel_time_ = this->my_world_->SimTime();
+#else
+    prev_update_time_ = last_cmd_vel_time_ = this->my_world_->GetSimTime();
```
Comment (jspricke) [inline src/gazebo_ros_diffdrive_uos.cpp]: Minor: fix indenting here.

File: src/gazebo_ros_diffdrive_uos.cpp [hunk 2]
```diff
@@ -243,10 +251,17 @@ void GazeboRosDiffdrive::UpdateChild()
   odom_vel_[0] = dr / step_time.Double();
   odom_vel_[1] = 0.0;
   odom_vel_[2] = da / step_time.Double();
-
```
Comment (jspricke) [inline src/gazebo_ros_diffdrive_uos.cpp]: Minor: leave empty line here.

**Review by jspricke — APPROVED** [review]
Just minor things, I'm fine with it otherwise.

**Comment by mintar** [comment]
LGTM

**Pull request merged**